frontend/src/autherror.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
```

Comment:
We might want to make this a teeny bit spiffier… I think the basic fact that there was an error should be in the page content and not just in the title of the tab/window.

Maybe something like:

```
<!DOCTYPE html>
<html>
  <head>
    <meta charset="UTF-8">
    <title>Error</title>
  </head>
  <body style="margin: 0; height: 100vh; width: 100vw; text-align: center;">
    <div style="height: 30vh;">
      &nbsp;
    </div>
    <div style="margin-left: auto; margin-right: auto">
      <h1>An Error Occurred</h1>
      <h3>Elevated permissions are required to see this content.</h3>
      <p>Please contact your system administrator for more information.</p>
    </div>
  </body>
</html>
```